packages/query/src/queries/get-current-template.graphql

```diff
@@ -126,5 +126,8 @@ query GetCurrentTemplate($uri: String!) {
 				...CoreVideoFrag
 			}
 		}
+		connectedNode {
+			id
+		}
```

Comment:
`templateByUri.is404` is now available on `snapwp-helper:develop` ([ref](https://github.com/rtCamp/snapwp-helper/pull/107)), so go ahead and update this PR to use it instead of `connectedNode`.